src/mantarray_desktop_app/main_process/queue_container.py

```diff
@@ -41,6 +41,8 @@ def __init__(self) -> None:
         self.from_server: queue.Queue[Dict[str, Any]] = queue.Queue()
         self.to_server: LightQueue = LightQueue()
 
+        self.from_websocket: queue.Queue[Dict[str, Any]] = queue.Queue()
```

Comment:
This needs to be a `LightQueue` like the other queue going to the websockets. We should probably also rename these queues now:
```
from_flask
to_websocket
from_websocket
```